cmd/cloud-controller-manager/app/controllermanager.go

```diff
@@ -148,7 +148,7 @@ func Run(s *options.CloudControllerManagerServer, cloud cloudprovider.Interface)
 			clientBuilder = rootClientBuilder
 		}
 
-		err := StartControllers(s, kubeconfig, rootClientBuilder, clientBuilder, stop, recorder, cloud)
+		err = StartControllers(s, kubeconfig, rootClientBuilder, clientBuilder, stop, recorder, cloud)
```

Comment:
The declaration is necessary and not duplicated, as its workspace is just the `run()` function. It is not recommended to reuse the `err` out of `run()`.